src/main/java/org/broadinstitute/consent/http/db/mapper/DatasetReducer.java

```diff
@@ -75,6 +79,25 @@ && hasColumn(rowView, "property_value", String.class)) {
         }
       }
     }
+
+    if (hasColumn(rowView, "fso_file_storage_object_id", Integer.class)
+      && Objects.nonNull(rowView.getColumn("fso_file_storage_object_id", Integer.class))
+    ) {
+      FileStorageObject fileStorageObject = rowView.getRow(FileStorageObject.class);
+
+      switch (fileStorageObject.getCategory()) {
```

Comment:
all it takes to add future files to the dataset would be to add a bit of logic into this switch case based upon the category of the file